lib/htsget-server/docker-compose.yml

```diff
@@ -29,5 +31,14 @@ services:
         - "traefik.docker.lbswarm=true"
         - "traefik.http.routers.htsget-app.rule=Host(`htsget-app.${CANDIG_DOMAIN}`)"
         - "traefik.http.services.htsget-app.loadbalancer.server.port=${HTSGET_APP_PORT}"
+    secrets:
+      - source: minio-access-key
+        target: access
+      - source: minio-secret-key
+        target: secret
+      - source: selfsigned-site-pem
```

Comment:
Why is ssl cert needed for htsget?

Reply:
If htsget is to access our self-signed minio instance (for dev setups), it needs the cert.

Reply:
It gets picked up by the correct version of htsget in its entrypoint.sh.